MonkeyBot/Common/CloudinaryCredentials.cs

```diff
@@ -2,8 +2,8 @@
 {
     public class CloudinaryCredentials
     {
-        public string Cloud { get; set; }
-        public string ApiKey { get; set; }
-        public string ApiSecret { get; set; }
+        public string? Cloud { get; set; }
+        public string? ApiKey { get; set; }
+        public string? ApiSecret { get; set; }
```

Comment:
Not sure if these should be nullable. It would probably be better to have a constructor that sets these properties and then they could also be readonly.

Reply:
I was thinking that too, but didn't want to mess too much with the code where these properties are assigned, without knowing exactly the original thought behind it.